packages/eslint-plugin/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## Unreleased -->
```

Comment:
Convention is to uncomment the unreleased heading when there is content underneath it

```suggestion
## Unreleased
```